sdk/storage/azure-storage-blobs/test/perf/inc/azure/storage/blobs/test/upload_blob_test.hpp

```diff
@@ -27,7 +27,7 @@ namespace Azure { namespace Storage { namespace Blobs { namespace Test {
    */
   class UploadBlob : public Azure::Storage::Blobs::Test::BlobsTest {
   private:
-    // C++ can upload and download from contiguos memory or file only
+    // C++ can upload and download from continuous memory or file only
```

Comment:
`contiguous` **is** a word and is appropriate here, it was previously misspelled as `contiguos`

Reply:
Fix in PR https://github.com/Azure/azure-sdk-for-cpp/pull/5220